js/src/hooks/useAdsCurrency.test.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react-hooks';
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import useAdsCurrency from './useAdsCurrency';
+import useStoreCurrency from './useStoreCurrency';
+
+// Force real timers, make sure all timers are actually ticking.
+jest.useRealTimers();
+
+jest.mock( '@wordpress/data-controls', () => ( {
+	apiFetch: jest.fn().mockName( 'apiFetch mock' ),
+} ) );
+jest.mock( './useStoreCurrency', () => ( {
+	__esModule: true,
+	default: jest.fn().mockName( 'useStoreCurrency' ).mockReturnValue( {
+		code: 'EUR',
+		precision: 3,
+		symbol: '€',
+		symbolPosition: 'right_space',
+		decimalSeparator: ';',
+		thousandSeparator: '|',
+		priceFormat: '%2$s %1$s',
+	} ),
+} ) );
+
+describe( 'useAdsCurrency', () => {
+	test( 'initially should return `{ currency: Object, hasFinishedResolution: undefined }`', () => {
+		const { result } = renderHook( () => useAdsCurrency() );
+
+		// assert initial state
+		expect( result.current ).toHaveProperty(
+			'hasFinishedResolution',
+			undefined
+		);
```

Comment:
❓ 
Although the `hasFinishedResolution` comes from another hook, the JSDoc of **useAdsCurrency** indicates it has only boolean type, but it actually returns `undefined` when initial. Could we make it consistent on either returned type or JSDoc for **useAdsCurrency**?

https://github.com/woocommerce/google-listings-and-ads/blob/f95a7cfd42b8de269e4a82dbd69a829c11c1a044/js/src/hooks/useAdsCurrency.js#L33